lib/demo_web/router.ex

```diff
@@ -23,4 +26,37 @@ defmodule DemoWeb.Router do
   # scope "/api", DemoWeb do
   #   pipe_through :api
   # end
+
+  ## Authentication routes
+
+  scope "/", DemoWeb do
+    pipe_through [:browser, :redirect_if_user_is_authenticated]
```

Comment:
Any plans to also support an api flow as well with something like a `--no-html` flag?

Reply:
Nope! @aaronrenner, we should probably check if `Phoenix.HTML` is available too before we generate. :)